components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/SQLConstants.java

```diff
@@ -245,6 +245,11 @@ public class SQLConstants {
             "WHERE UM_ORG_HIERARCHY.UM_PARENT_ID = :" + SQLPlaceholders.DB_SCHEMA_COLUMN_NAME_PARENT_ID +
             "; AND UM_ORG_HIERARCHY.DEPTH %s";
 
+    public static final String GET_CHILD_ORGANIZATION_IDS_WITH_RECURSIVE = "SELECT UM_ORG.UM_ID FROM UM_ORG " +
```

Comment:
```suggestion
    public static final String GET_CHILD_ORGANIZATION_IDS = "SELECT UM_ORG.UM_ID FROM UM_ORG " +
```

Reply:
We already have a variable named GET_CHILD_ORGANIZATION_IDS

Reply:
Used GET_IMMEDIATE_OR_ALL_CHILD_ORG_IDS instead
Resolved with e934897